src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/LoginPage.kt

```diff
@@ -19,9 +20,11 @@ class LoginPage(
             ExpectedConditions.presenceOfElementLocated(loginFormLocator)
         )
         val loginForm = driver.findElement(loginFormLocator)
-        loginForm.findElement(By.name("os_username")).sendKeys(user.name)
-        loginForm.findElement(By.name("os_password")).sendKeys(user.password)
+
+        driver.findElement(By.name("os_username")).sendKeysAndValidate(driver, user.name)
```

Comment:
Is there a reason for switching from `loginForm.findElement` to `driver.findElement`?